src/components/structures/RoomStatusBar.js

```diff
@@ -327,11 +327,18 @@ module.exports = React.createClass({
         } else {
             let consentError = null;
             let mauError = null;
+            const translateMauError = sub => {
+                if (mauError.data.admin_contact) {
+                    return <a href={mauError.data.admin_contact} target="_blank" rel="noopener">{sub}</a>;
```

Comment:
mailto?

Reply:
the admin contact should be a URI, so it should already have mailto: if it's an email address (also lets you specify a web form or something)